torchvision/transforms.py

```diff
@@ -214,6 +254,13 @@ def __init__(self, size):
         else:
             self.size = size
 
+    def get_params(self, img):
```

Comment:
I think those should be standalone functions, and not as part of the class object. Currently, there is no way for the user to obtain the crop parameters without instantiating the object, which is annoying.

Reply:
May be. Any ideas how we name them?

Reply:
Actually no, it doesn't make too much sense. These functions require params passed to `__init__`. Like self.size etc.

Reply:
The parameters passed into the `__init__` of the function can still be passed as arguments of the function, so I don't see the problem here. @alykhantejani what's your view on this matter?

Reply:
There are only 3/4 of them. So fine I guess. What do we name them?

Reply:
You can name them the same as their name in the constructor?